File: javascript/src/models/pandaDocConfig.ts
// this file is @generated

export interface PandaDocConfig {
  secret: string;
}

export const PandaDocConfigSerializer = {
  _fromJsonObject(object: any): PandaDocConfig {
    return {
      secret: object["secret"],
    };
  },

  _toJsonObject(self: PandaDocConfig): any {
    return {
      secret: self.secret,
    };
  },
};

File: rust/src/models/panda_doc_config.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
pub struct PandaDocConfig {
    pub secret: String,
}

impl PandaDocConfig {
    pub fn new(secret: String) -> Self {
        Self { secret }
    }
}
